WordPress/Classes/ViewRelated/Post/PostEditor.swift

```diff
@@ -25,6 +25,10 @@ protocol PostEditor: class, UIViewControllerTransitioningDelegate {
     ///
     var post: AbstractPost { get set }
 
+    /// If true, apply autosave content when the editor creates a revision.
+    ///
+    var shouldLoadAutosave: Bool { get set }
```

Comment:
You're right that the defining the setter in the protocol was a mistake. i.e. `{ get set }`.
I'd originally had `shouldLoadAutosave` outside of the protocol, and private - but went for the protocol to make sure all editors have the same interface. 

> In the case of shouldLoadAutosave, we should probably enforce it in the init() instead. But it's fine if we don't (or can't).

Do you mean removing the default value for `shouldLoadAutosave` in the `init` method?

I'd be happy to remove it from the protocol and keep it private.

Reply:
> Do you mean removing the default value for shouldLoadAutosave in the init method?

Something like adding the `init` in `PostEditor`

```swift
protocol PostEditor {
    init(
        post: AbstractPost,
        loadAutosaveRevision: Bool,
        replaceEditor: @escaping (EditorViewController, EditorViewController) -> (),
        editorSession: PostEditorAnalyticsSession?)
}
```

I'm not sure if this is worth it so I'm fine with leaving things as they are right now.

Reply:
> I'd originally had shouldLoadAutosave outside of the protocol, and private - but went for the protocol to make sure all editors have the same interface.
I misspoke here - I meant _behavior_ instead of _interface_.

I'm going to add the init method to the `PostEditor` like you mention above, both editors need a value for `loadAutosaveRevision` so it makes sense that it's in the initializer.